distribution/lib/Standard/Table/0.2.32-SNAPSHOT/src/Internal/Common_Helpers.enso

```diff
@@ -0,0 +1,21 @@
+from Standard.Base import all
+
+## PRIVATE
+
+   A helper used to select columns in both Standard.Table.Data.Table and Standard.Database.Data.Table
+
+   Arguments:
+   - columns: The current set of columns (must have a name property)
+   - column_list: The list of column names to filter down to
+   - reorder:
+select_columns : Vector -> Vector -> Boolean -> Vector
+select_columns = columns -> column_list -> (reorder=False) ->
+    mapper = x-> case x of
+        Text -> x
+        _ -> x.name
+    column_names = column_list.map mapper
```

Comment:
This is purely a subjective style thing - I would normally not give such a function a separate name, unless it is complicated enough that it deserves a type-annotation and/or a comment describing its usage or is used in multiple places.

For such simple situations, I'd suggest:
```suggestion
    column_names = column_list.map  x-> case x of
        Text -> x
        _ -> x.name
```

On the other hand - I do think that giving names can add value, but in this case I think the name would need to be more descriptive than `mapper` (which does not really say anything).

From me the rule of thumb would be:
1. Do I need to give it a name because it is used in many places? If yes, give it a name.
2. If not, does the function need a comment describing it? If yes, give it a descriptive name.
3. if not, can I give it a meaningful name that will hint at what the function is meant to do? If yes, give it a descriptive name.
4. If not, it may most likely stay anonymous.

But that is very much open to discussion.